src/components/Checkout/CheckoutForm.js

```diff
@@ -160,13 +169,257 @@ class CheckoutForm extends Component {
                               <div className="input-wrap country">
                                   <label className="required select-fallback">
                                       <span className="hide-content">Country</span>
-                                      <Field component="select" className="select-restyle" id="billing_country" required="required" name="billing_country">
+                                      <Field component="select" className="select-fallback" id="billing_country" required="required" name="billing_country">
                                           <option value>Country</option>
```

Comment:
For some reason this renders as `value="true"`, which means that the validation never triggers. Is there any way this option can be empty? (try adding `value=""` to see if that changes anything. (note this goes for the shipping country as well)